src/main/java/com/treasuredata/android/TreasureData.java

```diff
@@ -635,6 +639,19 @@ public String getErrorCode() {
         };
     }
 
+    private void appendDefaultValues(Map<String, Object> record, String database, String table) {
```

Comment:
Should we consider reverse the parameters' position for the signature to be consistent with `addEvent`?

Reply:
I agree, I changed the signature here to be consistent with addEvent() method. In iOS, the signature is the opposite (record first, then database and table). And I intend to keep default values method signature to be consistent too in td-ios-sdk side.